pkg/chart/v2/util/jsonschema.go

```diff
@@ -67,6 +69,7 @@ func ValidateAgainstSingleSchema(values Values, schemaJSON []byte) (reterr error
 	if err != nil {
 		return err
 	}
+	slog.Debug("unmarshalled JSON schema", "schema", schema)
```

Comment:
Did you want to output the JSON or the parsed Go object (struct)? This is the struct being printed. If you want the JSON itself, that would be `schemaJSON`.